args.c

```diff
@@ -94,8 +115,39 @@ void parse_args(int argc, char* argv[])
 			FTLfiles.log = "pihole-FTL.log";
 			FTLfiles.db = "pihole-FTL.db";
 			files.log = "pihole.log";
+			ok = true;
+		}
+
+		// List of implemented arguments
+		if(strcmp(argv[i], "-h") == 0 ||
+		   strcmp(argv[i], "help") == 0)
+		{
+			printf("pihole-FTL - The Pi-hole FTL engine\n\n");
+			printf("Usage:    sudo service pihole-FTL <action>\n");
+			printf("where '<action>' is one of start / stop / restart\n\n");
+			printf("Available arguments:\n");
+			printf("\td,  debug         More verbose logging,\n");
```

Comment:
Should the short form of `debug` have a hyphen before it, like the others do? `-d`